content/influxdb/v2.4/visualize-data/visualization-types/graph.md

```diff
@@ -31,12 +31,20 @@ the visualization dropdown.
 ###### Data
 - **X Column**: Select a column to display on the x-axis.
 - **Y Column**: Select a column to display on the y-axis.
-- **Time Format**: Select the time format. Options include:
-    {{< ui/timestamp-formats >}}
+{{% cloud-only %}}
+
+- **Adaptive Zoom**: Enable this option to zoom in on graphs for a more granular view. Zooming in dynamically updates the window period to re-query the data.
+  {{% warn %}}
+  If you've hard-coded the window period, we don't recommend enabling this option.
+  {{% /warn %}}
+
+{{% /cloud-only %}}
 
```

Comment:
@ALuckette -- this is the part with the adaptive zoom updates to focus on. The rest is just miscellaneous updates to match the UI per our conversation about improving the visualization docs.